<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkipDatabaseWidget</class>
 <widget class="QWidget" name="SkipDatabaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="removeItemPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="addItemPushButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="itemUpPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="go-up">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="itemDownPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="go-down">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="11">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Object Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Text</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Offset (hex)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Visible</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item row="2" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="10">
    <widget class="QPushButton" name="editItemPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="edit-select-all">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>removeItemPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SkipDatabaseWidget</receiver>
   <slot>onRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>SkipDatabaseWidget</receiver>
   <slot>onItemClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editItemPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SkipDatabaseWidget</receiver>
   <slot>onEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addItemPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SkipDatabaseWidget</receiver>
   <slot>onAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>SkipDatabaseWidget</receiver>
   <slot>onEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>itemUpPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SkipDatabaseWidget</receiver>
   <slot>onMoveItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>itemDownPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SkipDatabaseWidget</receiver>
   <slot>onMoveItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>SkipDatabaseWidget</receiver>
   <slot>onCurrentItemChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRemove()</slot>
  <slot>onItemClicked(QTreeWidgetItem*,int)</slot>
  <slot>onEdit()</slot>
  <slot>onAdd()</slot>
  <slot>onMoveItem()</slot>
  <slot>onCurrentItemChanged()</slot>
 </slots>
</ui>
